x-pack/plugins/security_solution/public/common/components/header_actions/header_actions.tsx

```diff
@@ -193,7 +193,13 @@ const HeaderActionsComponent: React.FC<HeaderActionProps> = ({
     [columnHeaders, dispatch, timelineId, defaultColumns]
   );
 
-  const ColumnSorting = useDataGridColumnSorting(myColumns, sortedColumns, {}, [], displayValues);
+  const ColumnSorting = useDataGridColumnSorting({
+    columns: myColumns,
+    sorting: sortedColumns,
+    schema: {},
```

Comment:
not that it wasn't like this before, but this and the array below should be a stable reference probably